src/etc/unicode.py

```diff
@@ -23,10 +23,7 @@
 # Since this should not require frequent updates, we just store this
 # out-of-line and check the unicode.rs file into git.
 
-import fileinput, re, os, sys, operator
-
-bytes_old = 0
-bytes_new = 0
+import fileinput, re, os, sys, operator, math
 
 preamble = '''// Copyright 2012-2016 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Does it matter? Also, this PR is merged, so if you want something to happen you should open a new issue or PR.